k8s/pkg/apis/longhorn/v1beta2/recurringjob.go

```diff
@@ -53,13 +53,19 @@ type RecurringJobSpec struct {
 	// The label of the snapshot/backup.
 	// +optional
 	Labels map[string]string `json:"labels,omitempty"`
+	// The parameters of the snapshot/backup.
```

Comment:
Can we add the parameters that are already supported in the description?

Reply:
fixed.